### PR TITLE
Add Symfony2 bundle

### PR DESCRIPTION
**Pull request by pierredup**

### CODE REVIEW

File: src/Namshi/Notificator/Symfony/DependencyInjection/NamshiNotificatorExtension.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+/**
+ * This is the class that loads and manages your bundle configuration
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
+ */
+class CSBillNotificationExtension extends Extension
```
Comment (odino) [inline src/Namshi/Notificator/Symfony/DependencyInjection/NamshiNotificatorExtension.php]: Why this class name? I dont understand where CSBill comes from :)

**Comment by odino** [comment]
thanks man, really appreciated! I just had one small doubt about the naming convention for the extension :)

cheers!

**Comment by pierredup** [comment]
Sorry I copy and pasted the code from a project where I use it, and just forgot to rename that class to the correct name :) Should be fixed now

**Comment by odino** [comment]
`4.0.2`

**Comment by odino** [comment]
thanks man!

**Pull request merged**